project2/style.css

```
@@ -0,0 +1,84 @@
+a {
+    color: #ffffff;
+    font-family: 'montserrat', sans-serif;
+    font-size: 15px;
+    text-align: center;
+
+}
+
+.card {
+    background-image: url(images/grand.jpg);
+    background-size: cover;
+    height: 277px;
+    position: relative;
+    width: 500px;
+
+}
+
+.date {
+    border: 0px;
+    font-size: 8px;
+    font-weight: normal;
+    margin-top: 5px;
+}
+
+div.transbox {
+    background: #e2a3a3;
+    height: 100%;
+    opacity: 0.8;
+    position: absolute;
+    right: 0;
+    top: 0;
+    width: 300px;
+}
+
+div.transbox:hover {
+    opacity: 1.0;
+}
+
+div.playbox {
+    border: 2px solid #ffffff;
+    font-weight: normal;
+    height: auto;
+    margin-top: 30px;
+    margin-left: 85px;
```

If you use this code instead, it will automatically horizontally center the button ✨🎉 :
```suggestion
    margin-left: auto;
    margin-right: auto;
```